api/routes/auth.py

```
@@ -265,23 +275,23 @@
             key="api_token",
             value=api_token,
             httponly=True,
-            secure=True
+            secure=_is_request_secure(request)
         )
         return response
 
     except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)